servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/GraphQLConfigurationProperties.kt

```diff
@@ -18,12 +18,15 @@ package com.expediagroup.graphql.server.spring
 
 import org.springframework.boot.context.properties.ConfigurationProperties
 import org.springframework.boot.context.properties.ConstructorBinding
+import org.springframework.core.Ordered
+import org.springframework.core.annotation.Order
 
 /**
  * [ConfigurationProperties] bean that defines supported GraphQL configuration options.
  */
 @ConstructorBinding
 @ConfigurationProperties("graphql")
+@Order(Ordered.HIGHEST_PRECEDENCE)
```

Comment:
does this have any effect?

Reply:
I think this got pulled in from some local debugging I was doing, I will remove